#!/usr/bin/env python

PACKAGE = 'oru_ipw_controller'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

#      Name                    Type      Lvl  Description                    Default   Min     Max
gen.add("drive_timeout",       double_t, 0,
        "Stop if no velocity command received within this time period [s]",  0.5,      0.01,    1)
gen.add("low_battery_level",   double_t, 0,
        "Voltage below which batteries are considered low.",                 18.0,     10,      22)

exit(gen.generate(PACKAGE, PACKAGE, "OruIpwController"))
